lib/client-assets.php

```diff
@@ -99,66 +99,84 @@ function gutenberg_register_scripts_and_styles() {
 
 	register_tinymce_scripts();
 
+	wp_register_script(
+		'wp-polyfill',
+		null,
```

Comment:
In one of my plugins, I was using this technique but I recall it was creating an issue for the script loading order. I can't reproduce thought here but thought I'd mention.

Reply:
Hmm, okay, it's good to note. I've not encountered an issue like that which you describe. I did find that `wp_add_inline_script` doesn't work with the `null` `$src`, which is why I've opted for the equivalent `wp_script_add_data` below.